src/main/java/hudson/plugins/sonar/SonarRunnerBuilder.java

```diff
@@ -376,7 +374,7 @@ private void addTaskArgument(ArgumentListBuilder args) {
 
   @VisibleForTesting
   void populateConfiguration(ExtendedArgumentListBuilder args, Run<?, ?> build, FilePath workspace,
```

Comment:
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png) The Cyclomatic Complexity of this method "populateConfiguration" is 11 which is greater than 10 authorized. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3AMethodCyclomaticComplexity)